src/JITModule.cpp

```diff
@@ -676,6 +678,13 @@ JITModule &make_module(llvm::Module *for_module, Target target,
             one_gpu.set_feature(Target::HVX_64);
             module_name = "hexagon";
             break;
+        case D3D12Compute:
+            one_gpu.set_feature(Target::D3D12Compute);
+            module_name = "d3d12compute";
+            #if !defined(_WIN32)
+                internal_error << "JIT support for Direct3D 12 is only implemented on Windows 10 and above.\n";
```

Comment:
Is there a way to actually check the windows version here easily?  Is minimum support Win10?

Reply:
Yes, Windows 10 is the minimum support:
https://support.microsoft.com/en-us/help/179113/how-to-install-the-latest-version-of-directx
(a notable exception would be XBoxOne)

I guess one way to check for D3D12 support would be to attempt to `LoadLibrary("d3d12.dll")` as it would only exist in computer running Windows 10.
What do you think?

Reply:
I think it's probably ok to do leave this as is; probably not worth trying to load the library here.